testing/src/main/graphql/betterbotz/products/BulkInsertProducts.graphql

```diff
@@ -0,0 +1,16 @@
+mutation BulkInsertProducts(
+    $values: [ProductsInput!]
+    $ifNotExists: Boolean
+    $options: MutationOptions
+) {
+    bulkInsertProducts(values: $values, ifNotExists: $ifNotExists, options: $options) {
+        applied
```

Comment:
so there is no notion of the array return type here.. I can simply say what I want from each value right?

Reply:
yes, the list return type is generated here: https://github.com/stargate/stargate/pull/874/files#diff-0abbca7fbce169d57be0fe97d3d62960e061a295d2d0974c5cc8702165416527R310-R311 
dynamically